### PR TITLE
Workflows: release tag and upload assets

### PR DESCRIPTION
**Pull request by RuffaloLavoisier**
image

This workflows,  just ONLY CREATE RELEASE AND UPLOAD ASSETS.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      - name: Compile language ${{ matrix.language }} model ${{ matrix.model }}
+        run: python3 .github/buildFirmware.py -l "${{ matrix.language }}" -m "${{ matrix.model }}" -b "${{ matrix.build-configuration }}"
+      - name: DEBUG LS-cmd
+        run: ls -lahR
+      - name: Release
+        uses: softprops/action-gh-release@v2
+        with:
+          files: "*.bin" # TEMP use zip to create an archive again
+          draft: false
+          prerelease: true
```
Comment (RuffaloLavoisier) [inline .github/workflows/release.yml:92]: I will only recommend drafts because the pre-release is posting, and the draft only saves. Is that correct?

Comment (simonmicro) [reply .github/workflows/release.yml:92]: I think drafts are indeed good, but I'll also recommend setting pre-release also to true by default.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
+name: Release OSW-OS Firmware
```
Comment (simonmicro) [inline .github/workflows/release.yml:1]: Maybe we can merge this new workflow into the existing one, which also builds all firmware variations already?

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        with:
+          python-version: "3.10"
+      - name: Install PlatformIO
+        run: python -m pip install --upgrade pip && pip install --upgrade platformio
+      - name: Rename config
+        run: mv include/config.h.example include/config.h
+      - name: Compile language ${{ matrix.language }} model ${{ matrix.model }}
+        run: python3 .github/buildFirmware.py -l "${{ matrix.language }}" -m "${{ matrix.model }}" -b "${{ matrix.build-configuration }}"
+      - name: DEBUG LS-cmd
+        run: ls -lahR
```
Comment (simonmicro) [inline .github/workflows/release.yml:86]: Please remove this

**Review by simonmicro — APPROVED** [review]

**Pull request merged**